tests/util/taikoHelper.js

```diff
@@ -20,7 +20,7 @@ async function repeatUntilNotVisible(element) {
     var isFound = true;
     do {
         try {
-            if (await element.exists(0, 0)) {
```

Comment:
This method looks like it was used in other places too. Why was this change needed?